modules/core/test/v2/integration/coins/bch.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * @prettier
+ */
+import * as should from 'should';
+import 'should-http';
+
+import { TestBitGo } from '../../../lib/test_bitgo';
+import * as nock from 'nock';
+
+nock.restore();
+
+describe('BCH:', function () {
+  let bitgo;
+  let wallet;
+
+  before(async function () {
+    bitgo = new TestBitGo({ env: 'test' });
+    bitgo.initializeTestVars();
+
+    await bitgo.authenticateTestUser(bitgo.testUserOTP());
+    wallet = await bitgo.coin('tbch').wallets().getWallet({ id: TestBitGo.V2.TEST_BCH_WALLET_ID });
+  });
+
+  describe('Send Transaction', function () {
+    it('should send fund to cashaddr recipient', async function () {
+      await bitgo.unlock({ otp: '0000000' });
+
+      const transaction = await wallet.send({
+        address: TestBitGo.V2.TEST_BCH_WALLET_CASH_ADDRESS,
+        amount: '5000',
+        walletPassphrase: TestBitGo.V2.TEST_BCH_WALLET_PASSPHRASE,
+        otp: bitgo.testUserOTP(),
+      });
+
+      should.exist(transaction);
+      transaction.should.have.property('transfer');
```

Comment:
it would probably be good to check the response has the correctly formatted cashaddr address too, but you can do that later if you want to come back to this.